typing/envaux.mli

```
@@ -18,13 +18,15 @@ open Format
 
 (* Convert environment summaries to environments *)
 
-val env_from_summary : Env.summary -> Subst.t -> Env.t
+val env_from_summary : allow_missing_modules:bool -> Env.summary -> Subst.t -> Env.t
 
 (* Empty the environment caches. To be called when load_path changes. *)
 
 val reset_cache: preserve_persistent_env:bool -> unit
 
-val env_of_only_summary : Env.t -> Env.t
+(* Reconstructs an environment from a summary. The Boolean [allow_missing_modules]
+   controls whether missing modules trigger an exception or fail silently. *)
```

I think "fail silently" is a little confusing - I believe more specifically what happens is the environment will just be missing the contents of opened modules if we can't find their cmi? We can probably just say that here.

---

I expanded the comment to clarify this and added a similar comment to the other function with the option to allow missing modules.